Makefile

```diff
@@ -152,9 +158,86 @@ unit-test: go-install
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanomodels_crd.yaml
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanobindings_crd.yaml
+	kubectl create -f vendor/${COH_PATH}/${CRD_PATH}/verrazzano.io_cohclusters_crd.yaml
+	kubectl create -f vendor/${HELIDON_PATH}/${CRD_PATH}/verrazzano.io_helidonapps_crd.yaml
+	kubectl create -f vendor/${WKO_PATH}/${CRD_PATH}/verrazzano.io_wlsoperators_crd.yaml
+	kubectl create -f vendor/${VMO_PATH}/${VMO_CRD_PATH}/verrazzano-monitoring-operator-crds.yaml
+
+	echo 'Deploy local cluster and required secret ...'
+	# Create the local cluster secret with empty kubeconfig data.  This will force the verrazzano-operator
+	# to use the in-cluster kubeconfig to access the managed cluster.
+	kubectl create secret generic verrazzano-managed-cluster-local --from-literal=kubeconfig=""
+	kubectl label secret verrazzano-managed-cluster-local k8s-app=verrazzano.oracle.com verrazzano.cluster=local
+	kubectl apply -f ${K8RESOURCE}/local-vmc.yaml
+
+	echo 'Load docker image for the verrazzano-operator...'
+	kind load docker-image --name ${CLUSTER_NAME} ${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+
+	echo 'Load docker image needed to install istio CRDs ...'
+	docker pull container-registry.oracle.com/olcne/istio_kubectl:1.4.6
+	docker image tag container-registry.oracle.com/olcne/istio_kubectl:1.4.6 container-registry.oracle.com/olcne/kubectl:1.4.6
+	kind load docker-image --name ${CLUSTER_NAME} container-registry.oracle.com/olcne/kubectl:1.4.6
+
+	echo 'Load docker image for the linux slim used for fake micro operators ...'
+	docker pull container-registry.oracle.com/os/oraclelinux:7-slim
+	kind load docker-image --name ${CLUSTER_NAME} container-registry.oracle.com/os/oraclelinux:7-slim
+
+	echo 'Install the istio CRDs ...'
+	kubectl create ns istio-system
+	kubectl apply -f ./test/k8resource/istio-crds.yaml
+	kubectl -n istio-system wait --for=condition=complete job --all --timeout=300s
+
+	echo 'Create verrazzano operator required secrets ...'
+	kubectl create namespace ${VERRAZZANO_NS}
+	kubectl create secret generic verrazzano -n ${VERRAZZANO_NS} \
+		--from-literal=password=admin --from-literal=username=admin
+
+	echo 'Deploy verrazzano operator  ...'
+	./test/create-deployment.sh ${DOCKER_IMAGE_NAME} ${DOCKER_IMAGE_TAG}
+	kubectl apply -f ${DEPLOY}/deployment.yaml
+
+	echo 'Run tests...'
+	# ginkgo -v --keepGoing -cover test/integ/... || IGNORE=FAILURE
+
+.PHONY: create-cluster
+create-cluster:
+ifdef JENKINS_URL
+	./build/scripts/cleanup.sh ${CLUSTER_NAME}
+endif
+	echo 'Create cluster...'
+	HTTP_PROXY="" HTTPS_PROXY="" http_proxy="" https_proxy="" time kind create cluster \
+		--name ${CLUSTER_NAME} \
+		--wait 5m \
+		--config=test/kind-config.yaml
+	kubectl config set-context kind-${CLUSTER_NAME}
+ifdef JENKINS_URL
+	# disabled this - not needed since we are not running from inside docker any more
```

Comment:
is there a need for this commented out line and comment

Reply:
The code is definitely needed to run in jenkins, but i will fix the comment